File: Java Script - Introduction to Programming/4.2. Complex Conditions - Exam Problems/05. Hotel Room.js
function hotelRoom([...arg]) {
  let [month, nights] = [...arg]
  let priceStudio = 50
  let priceApartment = 65

  switch (month) {
    case 'May':
    case 'October':
      priceStudio = 50 * nights
      priceApartment = 65 * nights

      if (nights > 14) {
        priceStudio *= 0.70;
        priceApartment *= 0.90
      } else if (nights > 7) {
        priceStudio *= 0.95
      }
      break

    case 'June':
    case 'September':
      priceStudio = 75.20 * nights
      priceApartment = 68.70 * nights

      if (nights > 14) {
        priceStudio *= 0.80
        priceApartment *= 0.90
      }
      break

    case 'July':
    case 'August':
      priceStudio = 76.00 * nights
      priceApartment = 77.00 * nights
      if (nights > 14) {
        priceApartment *= 0.90
      }
      break
  }

  return `Apartment: ${priceApartment.toFixed(2)} lv.\nStudio: ${priceStudio.toFixed(2)} lv.`
}

//console.log(hotelRoom(['May', 15]))

File: Pyrhon - Introduction to Programming/4.2. Complex Conditions - Exam Problems/05. Hotel Room.py
month, days = input(), int(input())
price_studio = 50
price_apartment = 65

if month == 'May' or month == 'October' :
    price_studio = 50 * days
    price_apartment = 65 * days

    if days > 14:
        price_studio *= 0.70
        price_apartment *= 0.90
    elif days > 7:
        price_studio *= 0.95

elif month == 'June' or month == 'September':
    price_studio = 75.20 * days
    price_apartment = 68.70 * days

    if days > 14:
        price_studio *= 0.80
        price_apartment *= 0.90

elif month == 'July' or month == 'August':
    price_studio = 76.00 * days
    price_apartment = 77.00 * days

    if days > 14:
        price_apartment *= 0.90

print(f'Apartment: {price_apartment:.2f} lv.\nStudio: {price_studio:.2f} lv.')



'''
June
14	
#Apartment: 961.80 lv.
#Studio: 1052.80 lv.
'''
